tensorflow/lite/micro/micro_profiler.h

```diff
@@ -61,6 +61,10 @@ class MicroProfiler {
   // Separated Value) form.
   void LogCsv() const;
 
+  // Prints the profiling information of each of the tags in CSV (Comma
+  // Separated Value) form.
```

Comment:
We should expand this description.

Something like:
```
// Prints  total ticks for each unique tag in CSV format.
//
// Output will have one row for each unique tag along with the total ticks summed across all events with that particular tag.
```

Reply:
Done. Thanks for the suggestion!